### PR TITLE
Fixed minor changes

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Removed some bugs in the code.

### CODE REVIEW

File: cloud_governance/cloud_resource_orchestration/clouds/aws/ec2/monitor_cro_instances.py [hunk 1]
```diff
@@ -77,7 +76,7 @@ def __monitor_instances(self):
         for ticket_id, cluster_data in cluster_tickets.items():
             for cluster_id, cluster_values in cluster_data.items():
                 cluster_values['cluster_name'] = cluster_id.split('/')[-1]
-                monitored_ticket_ids[ticket_id].append(cluster_values)
+                monitored_ticket_ids.setdefault(ticket_id, []).append(cluster_values)
```
Comment (ebattat) [inline cloud_governance/cloud_resource_orchestration/clouds/aws/ec2/monitor_cro_instances.py:79]: Is that ok, default is empty list ?
Why not to set it before the loop ?

Comment (athiruma) [reply cloud_governance/cloud_resource_orchestration/clouds/aws/ec2/monitor_cro_instances.py:79]: yes, we have two types of resources cluster and non-cluster. This loop is for cluster data. If we didn't find the non-cluster data default is empty. So we need to set the default as an empty list.

Comment (ebattat) [reply cloud_governance/cloud_resource_orchestration/clouds/aws/ec2/monitor_cro_instances.py:79]: ok

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**